src/Runner/StandardTestSuiteLoader.php

```diff
@@ -61,6 +61,12 @@ public function load(string $suiteClassFile): ReflectionClass
 
                     break;
                 }
+
+                if (stripos(substr($loadedClass, $offset - 1), '_' . $suiteClassName) === 0) {
```

Comment:
```suggestion
                // Support for PSR-0 namespaces (#5020)
                if (stripos(substr($loadedClass, $offset - 1), '_' . $suiteClassName) === 0) {
```

I would suggest adding a comment here, otherwise (at least speaking for myself) we will forget what this is for :wink:

Reply:
Yep, good idea. I pushed up parallel comments (7b49c195641e994f9d7ba5b21c12dccfdde8af56) for the parts that detect `\` and `_`.